packages/platform-android/native_modules.gradle

```diff
@@ -2,6 +2,7 @@ import groovy.json.JsonSlurper
 import org.gradle.initialization.DefaultSettings
 import org.apache.tools.ant.taskdefs.condition.Os
 
+def jsAppDir = buildscript.sourceFile.toString().split("node_modules/@react-native-community/cli-platform-android/native_modules.gradle")[0]
```

Comment:
nit: I think we can just split by `node_modules/@react-native-community/cli-platform-android`, which is a path to the module (the gradle file at the end is not needed and may break if we refactor files).

I was thinking to go with `node_modules`, but that would break in case dependencies were not hoisted.